pkg/apis/edgesite/v1alpha1/default.go

```diff
@@ -0,0 +1,166 @@
+					UpdateNodeWorkers:                 constants.DefaultUpdateNodeWorkers,
+				},
+			},
+			Edged: &edgecoreconfig.Edged{
+				Enable:                      true,
+				NodeStatusUpdateFrequency:   constants.DefaultNodeStatusUpdateFrequency,
+				DockerAddress:               constants.DefaultDockerAddress,
+				RuntimeType:                 constants.DefaultRuntimeType,
+				NodeIP:                      "",
+				ClusterDNS:                  "",
+				ClusterDomain:               "",
+				EdgedMemoryCapacity:         constants.DefaultEdgedMemoryCapacity,
+				RemoteRuntimeEndpoint:       constants.DefaultRemoteRuntimeEndpoint,
+				RemoteImageEndpoint:         constants.DefaultRemoteImageEndpoint,
+				PodSandboxImage:             constants.DefaultPodSandboxImage,
+				ImagePullProgressDeadline:   constants.DefaultImagePullProgressDeadline,
+				RuntimeRequestTimeout:       constants.DefaultRuntimeRequestTimeout,
+				HostnameOverride:            constants.DefaultHostnameOverride,
+				RegisterNodeNamespace:       constants.DefaultRegisterNodeNamespace,
+				InterfaceName:               constants.DefaultInterfaceName,
+				DevicePluginEnabled:         false,
+				GPUPluginEnabled:            false,
+				ImageGCHighThreshold:        constants.DefaultImageGCHighThreshold,
+				ImageGCLowThreshold:         constants.DefaultImageGCLowThreshold,
+				MaximumDeadContainersPerPod: constants.DefaultMaximumDeadContainersPerPod,
+				CGroupDriver:                edgecoreconfig.CGroupDriverCGroupFS,
+			},
+			MetaManager: &edgecoreconfig.MetaManager{
+				Enable:                true,
+				ContextSendGroup:      metaconfig.GroupNameEdgeController,
+				ContextSendModule:     metaconfig.ModuleNameEdgeController,
+				PodStatusSyncInterval: constants.DefaultPodStatusSyncInterval,
+			},
+		},
+	}
+}
+
+// NewMinEdgeSiteConfig returns a common EdgeSiteConfig object
+func NewMinEdgeSiteConfig() *EdgeSiteConfig {
+	return &EdgeSiteConfig{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       Kind,
+			APIVersion: path.Join(GroupName, APIVersion),
+		},
+		DataBase: &edgecoreconfig.DataBase{
+			DriverName: DataBaseDriverName,
+			AliasName:  DataBaseAliasName,
+			DataSource: DataBaseDataSource,
+		},
+		EdgeSite: &EdgeSiteDetail{
+			NodeName: "",
+		},
+		KubeAPIConfig: &cloudcoreconfig.KubeAPIConfig{
+			Master:     "",
+			KubeConfig: constants.DefaultKubeConfig,
+		},
+		Modules: &Modules{
+			Edged: &edgecoreconfig.Edged{
+				DockerAddress:         constants.DefaultDockerAddress,
+				RuntimeType:           constants.DefaultRuntimeType,
+				NodeIP:                "",
+				ClusterDNS:            "",
+				ClusterDomain:         "",
+				RemoteRuntimeEndpoint: constants.DefaultRemoteRuntimeEndpoint,
+				RemoteImageEndpoint:   constants.DefaultRemoteImageEndpoint,
+				PodSandboxImage:       constants.DefaultPodSandboxImage,
+				HostnameOverride:      constants.DefaultHostnameOverride,
+				RegisterNodeNamespace: constants.DefaultRegisterNodeNamespace,
```

Comment:
No need to show RegisterNodeNamespace in min config, as we are going to deprecate RegisterNodeNamespace in a near future
```suggestion
```

Reply:
done